src/main/java/org/folio/rs/domain/entity/PlainMappingEntity.java

```diff
@@ -0,0 +1,27 @@
+package org.folio.rs.domain.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor(staticName = "of")
+@NoArgsConstructor
+@Entity
+@Table(name = "location_mappings")
+public class PlainMappingEntity {
```

Comment:
name = "location_mappings" is used already for FullMappingEntity, is it something that expected.

Reply:
Yes, both these entities use same table, FullMappingEntity additionally uses original_lovations table via one-to-many relation.